http-add-on/templates/interceptor/deployment.yaml

```diff
@@ -59,7 +59,7 @@ spec:
         - name: KEDIFY_EXTERNAL_SCALER_SERVICE_NAME
           value: "{{ .Chart.Name }}-{{ .Values.scaler.service }}"
         - name: KEDIFY_XDS_MAX_CONCURRENT_STREAMS
-          value: "{{ .Values.interceptor.xds.maxConcurrentStreams }}"
+          value: "{{ .Values.interceptor.xds.maxConcurrentStreams | int64 }}"
```

Comment:
without this, the default value got rendered as "1e+06" and interceptor was crashlooping with
```
panic: envconfig.Process: assigning KEDIFY_XDS_MAX_CONCURRENT_STREAMS to XDSMaxConcurrentStreams: converting '1e+06' to type uint32. details: strconv.ParseUint: parsing "1e+06": invalid syntax

goroutine 1 [running]:
github.com/kelseyhightower/envconfig.MustProcess(...)
        github.com/kelseyhightower/envconfig@v1.4.0/envconfig.go:233
github.com/kedacore/http-add-on/interceptor/config.MustParseKedify()
        github.com/kedacore/http-add-on/interceptor/config/kedify.go:85 +0x99
main.main()
        github.com/kedacore/http-add-on/interceptor/main.go:62 +0x9f
```
not sure how it got past the PR checks